ext/js/pages/settings/profile-controller.js

```diff
@@ -261,6 +261,10 @@ export class ProfileController {
         // Update profile index
         if (settingsProfileIndex === profileIndex) {
```

Comment:
```suggestion
        if (settingsProfileIndex >= profileIndex) {
```
Isn't this math accounted for above? Line 231

```
if (profileCurrentNew >= profileIndex) {
            profileCurrentNew = Math.min(profileCurrentNew - 1, this._profiles.length - 1);
            modifications.push({
                action: 'set',
                path: 'profileCurrent',
                value: profileCurrentNew,
            });
        }
        
```